src/node_api.h

```diff
@@ -73,16 +78,39 @@ typedef struct napi_module {
   static void fn(void)
 #endif
 
+#ifdef NAPI_EXPERIMENTAL
+#ifdef NODE_API_CUSTOM_FEATURES
+
+// Define value of node_api_module_features variable in your module when
+// NODE_API_CUSTOM_FEATURES is set in gyp file.
+extern node_api_features node_api_module_features;
+#define NODE_API_DEFINE_DEFAULT_FEATURES
+
+#else  // NODE_API_CUSTOM_FEATURES
+
+#define NODE_API_DEFINE_DEFAULT_FEATURES                                       \
+  static node_api_features node_api_module_features = node_api_default_features;
```

Comment:
Comments as before, I think this will end up setting to be specific to the Node.js version used to build versus the Node-API version selected. Should discuss this more.

Reply:
I am not sure how to address this. Let's discuss it in the Node-API meeting. The concern as I understand it is that if we add more experimental features, we may fail to propagate them to the previous versions of Node.JS, and thus it may cause differences in the behavior between Node.JS versions, while we have the same version of Node-API, right?